test/global-rate-limit.test.js

```diff
@@ -10,6 +10,7 @@ const noop = () => { }
 const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms))
 
 const REDIS_HOST = '127.0.0.1'
+const isSkipRedis = Boolean(process.env.SKIP_REDIS)
```

Comment:
this code is not needed, please remove it and commit using `git commit -n -m 'remove skip redis'` to skip the git hook

Reply:
okay, done